src/mobility/components/CarSharingStationBottomSheet.tsx

```diff
@@ -48,6 +49,21 @@ export const CarSharingStationSheet = ({stationId, distance, close}: Props) => {
     rentalAppUri,
   });
   const stationName = useTextForLanguage(station?.name.translation) ?? '';
+  const analytics = useAnalytics();
+
+  useEffect(() => {
+    analytics.logEvent('Mobility', 'Car sharing station selected', {
+      id: station?.id,
```

Comment:
Is `station?.id` the same as `stationId` in that case, we could use that value instead to avoid null values in case.

Reply:
It is true for stationId, yes. I'll change that.

It might be tempting to use `operatorName` from the above hook as well, but this is computed in a separate tick and if `operatorName` is included in the `useEffect` dependencies it will cause the event to be logged twice.